cmake/PACE_Docs.cmake

```diff
@@ -0,0 +1,65 @@
+# Build documentation
+
+set(DOCS_DIR "${Horace_ROOT}/documentation/user_docs" CACHE STRING "Directory containing user documentation")
```

Comment:
Does this need to be a cache variable? I'd usually only make a variable a cache variable if I wanted to make it configurable